source/beamline/BioXAS/BioXASSideBeamline.cpp

```diff
@@ -208,7 +208,7 @@ void BioXASSideBeamline::setupComponents()
 
 	// Endstation safety shutter.
 
-	endstationShutter_ = new  BioXASEndstationShutter("BioXASSideEndstationShutter", "SSH1607-5-I22-01", this);
+	endstationShutter_ = new  CLSExclusiveStatesControl("SSH1607-5-I22-01", "SSH1607-5-I22-01:state", "SSH1607-5-I22-01:opr:open", "SSH1607-5-I22-01:opr:close", this);
```

Comment:
@helfrij I saw you add them to the BioXASShutters. 

What I mean is whether ` endstationShutter_` and ` frontEndShutter_` should be instanced in `BioXASShutters` and be deleted from the `BioXASSideBeamline`?